drivers/wifi/nxp/Kconfig.nxp

```diff
@@ -469,6 +469,12 @@ config NXP_WIFI_11R
 	help
```

Comment:
Please change the commit subject of `drivers: add ...` to `drivers: wifi: nxp: ....` so that it is clear what type of driver we are changing and which vendor.

Reply:
Updated, thanks